NDB.Covid19/NDB.Covid19.iOS/Views/Welcome/ChildViews/WelcomePageFourViewController.cs

```diff
@@ -21,6 +21,19 @@ public override void ViewDidLoad()
             BackArrow.AccessibilityLabel = SettingsViewModel.SETTINGS_CHILD_PAGE_ACCESSIBILITY_BACK_BUTTON;
         }
 
+        public override void ViewDidAppear(bool animated)
+        {
+            base.ViewDidAppear(animated);
+            foreach (var v in View.Subviews)
```

Comment:
create a separate method in StyleUtils as this is duplicated code everywhere. and probably we will need it also in another views later

Reply:
Fixed